src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/task/Warning.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.task;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@ApiModel(
+    value = "Warning",
+    description = "Warning object containing the field to sort on and the order"
+)
+@EqualsAndHashCode
+@ToString
+public class Warning {
+    @ApiModelProperty(
+        notes = "A code that distinguishes which Warning is to be applied ")
+    private String warningCode;
+
+    @ApiModelProperty(
+        notes = "Text associated to the warning code that can be shown in the UI")
+    private String warningText;
+
+    public Warning() {
+        // noop -- for deserializer
+        super();
```

Comment:
you can delete the super() here

Reply:
actually since you are using a jsoncreator constructor you can delete this empty constructor